core/runtime/src/main/java/io/quarkus/runtime/LaunchMode.java

```diff
@@ -24,7 +24,7 @@ public boolean isDevOrTest() {
      * Returns true if the current launch is the server side of remote dev.
      */
     public static boolean isRemoteDev() {
-        return (current() == DEVELOPMENT) && "true".equals(System.getenv("QUARKUS_LAUNCH_DEVMODE"));
+        return (current() == DEVELOPMENT) && System.getenv("QUARKUS_LAUNCH_DEVMODE").equals("true");
```

Comment:
You shouldn't change this. It looks odd but this is actually the safer pattern as it works even if `System.getenv("QUARKUS_LAUNCH_DEVMODE")` is `null`. Your new pattern will fail with a NPE in this case.

You know it once you got bitten by it :).